bench.py

```diff
@@ -2,12 +2,45 @@
 import os
 import sys
 import time
+import psutil
+import json
+import urllib
+import urllib2
+from datetime import datetime
 
 import click
 import statistics
+import scrapy
 
 
-def calculator(test, arg, n_runs, only_result, workpath=os.getcwd()):
+CODESPEED_URL = 'http://localhost:8000/'
```

Comment:
Would be nice to pass all configuration via the command line options. Is it possible to defined such common options '--n-runs', '--only_result',  '--uploadresult', etc.) just once? After all, they are common for all benchmarks? Or make it a single click command and make benchmark name it's argument?